web_services/foxx/brightid/db.js

```diff
@@ -545,9 +568,15 @@ function getLastContextIds(coll, verification) {
 }
 
 function userVerifications(user) {
-  return verificationsColl.byExample({
+  const verifications = verificationsColl.byExample({
     user
-  }).toArray().map(v => v.name);
+  }).toArray();
+  return Object.assign({}, ...verifications.map(v => ({[v.name]: v})));
```

Comment:
You can use:
`return _.keyBy(verifications, v => v.name);`

Reply:
It's done
https://github.com/BrightID/BrightID-Node/commit/c43685574cdc79a6bc34aa99cb5510425cc82a57